controllers/sr_controller/sr/robot/sensor_devices.py

```diff
@@ -1,13 +1,19 @@
+from abc import ABC, abstractmethod
+
 from sr.robot.utils import map_to_range
 from sr.robot.settings import TIME_STEP
 from sr.robot.randomizer import add_jitter
 
 
-class SensorBase(object):
+class SensorBase(ABC):
 
+    @abstractmethod
     def __init__(self, webot, sensor_name):
-        self.webot = webot
-        self.sensor_name = sensor_name
+        pass
```

Comment:
Is having this `__init__` in an `ABC` actually useful? I've not used ABCs much, but this doesn't feel to me like it's adding any value.

Reply:
Having used `ABC` a lot in `j5`, I'm pretty confident that `@abstractmethod` has zero effect.

Reply:
That's odd. My understanding was that the decorator helped prevent you not implementing a method.
For example the following code:
``` python
import abc

class B(abc.ABC):
    def foo(self):
        print('foo')

    @abc.abstractmethod
    def bar(self):
        print('bar')


class C(B):
    def bar(self):
        print('C.bar')

class D(B):
    pass


C()
D()
```

errors like this:
```
Traceback (most recent call last):
  File "Documents\bees.py", line 21, in <module>
    D()
TypeError: Can't instantiate abstract class D with abstract methods bar
```

This is a bit orthogonal though -- my original point was more that (an empty) `__init__` specifically in an ABC doesn't add anything because all classes have one _anyway_.